client/src/theme/index.js

```diff
@@ -37,23 +32,23 @@ let theme = createTheme({
         {
           props: { variant: 'secondary' },
           style: {
-            border: '3px solid #00008B',
+            border: `3px solid ${uiKitColors.secondary}`,
```

Comment:
Into it. I'll do that now.

I'm a vote to move towards ui kit components that aren't tied to the theme eventually.

eg exporting a list of ui components:

`<ButtonSecondary />` instead of `<Button variant="secondary">`